application/testing/CMakeLists.txt

```diff
@@ -194,6 +194,15 @@ f3d_test(NAME TestUpDirectionNoSign DATA suzanne.ply ARGS --up=X DEFAULT_LIGHTS)
 f3d_test(NAME TestTextureMatCap DATA suzanne.ply ARGS --texture-matcap=${F3D_SOURCE_DIR}/testing/data/skin.png DEFAULT_LIGHTS)
 f3d_test(NAME TestTextureMatCapWithTCoords DATA WaterBottle.glb ARGS --geometry-only --texture-matcap=${F3D_SOURCE_DIR}/testing/data/skin.png DEFAULT_LIGHTS)
 
+if(VTK_VERSION VERSION_GREATER_EQUAL 9.3.20231108)
```

Comment:
which MR does this requires ?

Reply:
SSBO support: https://gitlab.kitware.com/vtk/vtk/-/merge_requests/10675 I suppose ?

Reply:
Indeed. I'll add a comment